webviz_subsurface/_utils/simulation_timeseries.py

```diff
@@ -36,3 +37,104 @@ def get_simulation_line_shape(
     except (AttributeError, KeyError):
         pass
     return line_shape_fallback
+
+
+def calc_series_statistics(df, vectors, refaxis="DATE"):
+    """Calculate statistics for given vectors over the ensembles
+    refaxis is used if another column than DATE should be used to groupby.
+    """
+    # Invert p10 and p90 due to oil industry convention.
```

Comment:
Since it is "regular" percentiles as of now, maybe we should wait with reverting it? Instead we can change the labels from `P10` and `P90` to `Low (P10)` and `High (P90)` respectively, to improve user communication.

Arguments:
- It is useful to add `Low`/`High` prefixes regardless.
- We don't change current behavior.
- The oil industry is not completely agreeing on a standard. In Equinor, D&W and economical analysis tend to use the common worldwide definition. Since webviz is aiming at being cross disciplinary, maybe the worldwide definition, with clear prefixes, is the best/safest compromise?